404.html

```diff
@@ -1,25 +1,32 @@
 ---
 layout: default
-title: 404
+title: 404 — Page not found
+body_class: notfound
+add_css: page404.css
 ---
 
-<style type="text/css" media="screen">
-  .container {
-    margin: 10px auto;
-    max-width: 600px;
-    text-align: center;
-  }
-  h1 {
-    margin: 30px 0;
-    font-size: 4em;
-    line-height: 1;
-    letter-spacing: -1px;
-  }
-</style>
-
-<div class="container">
-  <h1>404</h1>
-
-  <p><strong>Page not found :(</strong></p>
-  <p>The requested page could not be found.</p>
+<div class="pagehead bg{{ '1,2,3,4,5,6' | split: ',' | sample }}">
```

Comment:
Does this randomly choose a new background at build time? Why would we want this?

Reply:
Jonathan would like to have different backgrounds so we thought of random backgrounds. But we could also define each page a specific background. 
I can update.

Reply:
I'm just saying, they're not really random - it would only ever update when we rebuild the website?